frontend/src/views/Layout/DashboardLayout_gdd.vue

```diff
@@ -94,15 +94,15 @@ export default {
     async loadBalance() {
       const result = await communityAPI.balance(this.$store.state.session_id)
       if (result.success) {
-        this.balance = result.result.data.balance / 10000
+        this.balance = result.result.data.balance// / 10000
       } else {
         // what to do when loading balance fails?
       }
     },
     async loadGDTBalance() {
       const result = await communityAPI.transactions(this.$store.state.session_id)
       if (result.success) {
-        this.GdtBalance = result.result.data.gdtSum / 10000
+        this.GdtBalance = result.result.data.gdtSum// / 10000
```

Comment:
Is this the expected behavior of GDT as well? Cannot test...

Reply:
gdt cames now also as float